app/controllers/api/v1/book_suggestions_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class BookSuggestionsController < ApiController
+      skip_before_action :authenticate_request
+
+      def create
+        @book_suggestion = BookSuggestion.new(book_suggestion_params)
+        @book_suggestion.user_id = current_user.try(:id) || nil
```

Comment:
`current_user.try(:id)` is going to return `nil` if current_user is not present, no need for `|| nil`